setup.py

```diff
@@ -18,10 +18,13 @@ def download_unpack(rewrite=False):
 		with open('libtorch.zip', 'wb') as f:
 			f.write(datatowrite)
 
-	if (not Path('GodotModule/libtorch').exists()) or rewrite:
```

Comment:
What happened to rewrite? As it is always `True` I guess `def download_unpack(` should not have it anymore.

Reply:
`rewrite` will still default to `false`. Previously this meant that `download_unpack()` would not download or unpack the zip if it is already present. I changed the script so that it would always unpack, no matter what value `rewrite` is. Setting `rewrite` to `true` will still make the script download the zip file again.